single_include/catch.hpp

```diff
@@ -6817,7 +6851,8 @@ namespace {
         static Win32ColourImpl s_instance;
         return &s_instance;
     }
-
+#else
```

Comment:
No, just weird part of coding convention I do not always follow.

On 8 January 2016 at 13:51, Jędrzej <notifications@github.com> wrote:

> In single_include/catch.hpp
> <https://github.com/philsquared/Catch/pull/570#discussion_r49185040>:
>
> > @@ -6817,7 +6851,8 @@ namespace {
> >          static Win32ColourImpl s_instance;
> >          return &s_instance;
> >      }
> > -
> > +#else
>
> Empty #else clause any reason it's there?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/philsquared/Catch/pull/570/files#r49185040>.
>



-- 
Best Regards,
Vladimir Stepanov